### DISCUSSION TITLE
RAG does not work for new users using AzureOpenID as a signup/login method.

### DISCUSSION THREAD

**Discussion by RAPHCVR** [danny-avila/LibreChat#4035] [Troubleshooting]
### What happened?

The users of our instance have an error, not logged anywhere, but shown in the webapp, saying an error happened when processing the file, I am using mongodb Atlas. And OpenID with AzureEntraID. Upload of files works fine with local users, but when signing up using Entra ID, the RAG function does not work.

Is that a mongodb database thing ?

### Steps to Reproduce

1. Setting Up LibreChat
2. Using Azure Entra ID according to the LibreChat Doc
3. signing up with entra ID, using the openid log in button

### Relevant log output

```shell
Here is maybe relevant .env fields:

# OpenID
OPENID_CLIENT_ID=[sensitive]
OPENID_CLIENT_SECRET=[sensitive]
OPENID_ISSUER=h[sensitive]
OPENID_SESSION_[sensitive]
OPENID_SCOPE="openid profile email"
OPENID_CALLBACK_URL=/oauth/openid/callback
OPENID_REQUIRED_ROLE=
OPENID_REQUIRED_ROLE_TOKEN_KIND=
OPENID_REQUIRED_ROLE_PARAMETER_PATH=

OPENID_BUTTON_LABEL=Azure Team
OPENID_IMAGE_URL=[sensitive]
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
This is not the case for others, also there isn't enough logs here to see what's going on. Check your ./logs directory at project root, in particular the most recent file(s) with "error" in the name after this happens.

**Reply by RAPHCVR** [reply]
{"level":"error","message":"[handleAbortError] AI response error; aborting request: Azure OpenAI API instance name not >
{"level":"error","message":"[openidStrategy] downloadImage: Error downloading image at URL \"https://graph.microsoft.co>
these are the only errors I can see, on logs of docker or on the error log file.

**Reply by RAPHCVR** [reply]
But it does not seems to have something to do with the rag thing

**Comment by RAPHCVR** [comment]
> This is not the case for others, also there isn't enough logs here to see what's going on. Check your ./logs directory at project root, in particular the most recent file(s) with "error" in the name after this happens.

@danny-avila some ideas? (After reading my answers below your message)
